libs/vscode/generate-ui-webview/src/lib/init-generate-ui-webview.ts

```diff
@@ -66,7 +67,7 @@ function parseIntoPrefillValues(
   yargsParseResult: Awaited<ReturnType<typeof yargs.parse>>,
   options: Option[]
 ): Record<string, string> {
-  const positionals = yargsParseResult['_'].slice(1);
+  const positionals = yargsParseResult['_'].slice(2);
```

Comment:
do we have standard code for parsing positionals in our devkit or is this the best way we can go for?